arbitrum/recordingdb.go

```diff
@@ -57,10 +58,10 @@ func (db *RecordingKV) Get(key []byte) ([]byte, error) {
 		res, err = db.diskDb.Get(key)
 	} else if ok, _ := rawdb.IsActivatedAsmKey(key); ok {
 		// Arbitrum: the asm is non-consensus
-		return db.diskDb.Get(key)
+		return db.wasmDb.Get(key)
 	} else if ok, _ := rawdb.IsActivatedModuleKey(key); ok {
 		// Arbitrum: the module is non-consensus (only its hash is)
-		return db.diskDb.Get(key)
+		return db.wasmDb.Get(key)
```

Comment:
Will these ever be called? Shouldn't they be called on the wasm database instead?

Reply:
moved some things around and now it is indeed not needed